### PR TITLE
* FIX: pctr_score_companies()` no longer drops companies depending on the number of rows of co2 data

### LINKED ISSUE #122: Debug edge cases of 'pstr_score_companies()'
We have tests that characterize bugs. It's now time to
fix them.

### PR DESCRIPTION
**Pull request by maurolepore**
Closes #122 

This PR replaces `inner_join()` with `left_join()` to preserve all companies.

See `?dplyr::inner_join()`:

> An inner_join() only keeps observations from x that have a matching key in y.

> The most important property of an inner join is that unmatched rows in either input are not included in the result. This means that generally inner joins are not appropriate in most analyses, because it is too easy to lose observations.

### CODE REVIEW

File: R/pctr_score_companies.R [hunk 1]
```diff
@@ -41,7 +41,7 @@ pctr_score_companies <- function(scored_activities, companies) {
 
   # join by activity_product_uuid and other joint columns from companies with scored_activities
   companies_scores <- companies |>
-    inner_join(scored_activities, by = c("activity_product_uuid", "ei_activity", "unit"))
+    left_join(scored_activities, by = c("activity_product_uuid", "ei_activity", "unit"))
```
Comment (maurolepore) [inline R/pctr_score_companies.R:44]: Here is what I assume is a bug.

Comment (maurolepore) [reply R/pctr_score_companies.R:44]: It would be best for the author @Tilmon to confirm, but it seems like a reasonable assumtion.

From `?dplyr::inner_join()`:

> An inner_join() only keeps observations from x that have a matching key in y.
> The most important property of an inner join is that unmatched rows in either input are not included in the result. This means that generally inner joins are not appropriate in most analyses, because it is too easy to lose observations.
